e2e-tests/cypress/tests/integration/playbooks/channels/broadcast_spec.js

```diff
@@ -367,7 +367,7 @@ const deleteLatestPostRoot = (testTeam, channelName) => {
             });
 
         // * Post extra options is visible
-        cy.findByLabelText('Post extra options').should('exist');
+        cy.findByLabelText('Post extra options').should('have.attr', 'role', 'menu').and('exist');
```

Comment:
I'm reviewing the rest of the code now, but this test failed in the last E2E run: https://automation-dashboard.vercel.app/cycle/12635348539_1-5746019-pr-onprem-ent